spring-cloud-contract-stub-runner/src/test/java/org/springframework/cloud/contract/stubrunner/GitStubDownloaderTests.java

```diff
@@ -86,12 +86,13 @@ public void should_return_a_null_downloader_for_a_non_git_repo() {
 	public void should_pick_stubs_for_group_and_artifact_with_version_from_a_git_repo()
 			throws Exception {
 		StubDownloaderBuilder stubDownloaderBuilder = new ScmStubDownloaderBuilder();
+		String contractFolderLocation = (file("/git_samples/contract-git/").getAbsolutePath()
```

Comment:
in these tests made sure the location is all git://a/b/c/ else was resolving to git://C:\a\b\c/ and was messing up the clone